source/ingress.go

```diff
@@ -71,7 +71,8 @@ func (sc *ingressSource) Endpoints() ([]*endpoint.Endpoint, error) {
 	for _, ing := range ingresses.Items {
 		// Check controller annotation to see if we are responsible.
 		controller, ok := ing.Annotations[controllerAnnotationKey]
-		if ok && controller != controllerAnnotationValue { //TODO(ideahitme): log the skip
+		if ok && controller != controllerAnnotationValue {
+			log.Debugf("skipping ingress %s/%s another controller annotation is found: %s", ing.Namespace, ing.Name, controller)
```

Comment:
"is found" also sounds weird to me: the annotation is not found, but the value does not match **our** controller